entrypoint-monitoring/main.tf

```diff
@@ -58,7 +58,8 @@ data "aws_iam_policy_document" "monitor_inline_policy" {
 }
 
 module "monitor_lambda" {
-  source                 = "github.com/massgov/mds-terraform-common//lambda?ref=1.0.26"
+  # TODO: change branch to tag before deploying
+  source                 = "github.com/massgov/mds-terraform-common//lambda?ref=DP-27416-terraform-upgrade-0.13"
```

Comment:
I think using `ref=1.0.26` was an artifact of ds-infra not supporting the `ephemeral_storage` feature introduced in `lambda@1.0.27`. Since the lambda module will now require a provider version that supports `ephemeral_storage`, I think we may just want to reference the local module